argo_rollouts/assets/logs/argo_rollouts.yaml

```diff
@@ -0,0 +1,93 @@
+id: argo_rollouts
+metric_id: argo-rollouts
+backend_only: false
+facets:
+  - groups:
+      - Argo Rollouts
+    name: Argo Resource Name
+    path: argo_rollouts.resource.name
+    source: log
+  - groups:
+      - Argo Rollouts
+    name: Argo Rollouts Name
+    path: argo_rollouts.name
+    source: log
+pipeline:
+  type: pipeline
+  name: Argo Rollouts
+  enabled: true
+  filter:
+    query: source:argo_rollouts
+  processors:
+    - type: grok-parser
+      name: Parsing Argo Rollouts logs
+      enabled: true
+      source: message
+      samples:
+        - time="2024-03-28T20:36:30Z" level=info msg="successfully acquired lease argo-rollouts/argo-rollouts-controller-lock"
+        - time="2024-02-20T20:49:32Z" level=info msg="Start processing" resource=default/rollouts-demo
+        - time="2024-02-20T20:49:32Z" level=info msg="Processing completed" resource=default/rollouts-demo
+        - time="2024-02-20T20:49:32Z" level=info msg="Started syncing rollout" generation=4 namespace=default resourceVersion=24435167 rollout=rollouts-demo
+        - time="2024-02-20T20:49:32Z" level=info msg="No TrafficRouting Reconcilers found" namespace=default rollout=rollouts-demo
+      grok:
+        matchRules: |
+          argo_common time="%{_date}" level=%{_level} msg="%{_msg}" ?%{data::keyvalue("=","/:")}
+        supportRules: |
+          _date %{date("yyyy-MM-dd'T'HH:mm:ssZ"):timestamp}
+          _msg %{data:msg}
+          _level %{word:level}
+    - type: attribute-remapper
+      name: Map `rollout` to `argo_rollouts.name`
+      enabled: true
+      sources:
+        - rollout
+      target: argo_rollouts.name
+      preserveSource: false
+      overrideOnConflict: false
+      sourceType: attribute
+      targetType: attribute
+    - type: attribute-remapper
+      name: Map `resource` to `argo_rollouts.resource.name`
+      enabled: true
+      sources:
+        - resource
+      target: argo_rollouts.resource.name
+      preserveSource: false
+      overrideOnConflict: false
+      sourceType: attribute
+      targetType: attribute
+    - type: attribute-remapper
+      name: Map `namespace` to `argo_rollouts_namespace`
+      enabled: true
+      sources:
+        - namespace
+      target: argo_rollouts_namespace
```

Comment:
I think the reason I did that specifically was because I wanted it to match the tag in the metrics. The metrics come in with a tag called `argo_rollouts_namespace` so I figured I can map the attribute there for better correlation and searching in dashboards with template variable `$argo_rollouts_namespace` that way we can filter metrics by that tag and then also use `@$argo_rollouts_namespace` to also filter for the correct logs. 

Dashboard link here:
https://dd.datad0g.com/dashboard/c5v-k7x-n4n/argo-rollouts-overview?fromUser=true&refresh_mode=sliding&tpl_var_argo_rollouts_namespace%5B0%5D=default&view=spans&from_ts=1711635981944&to_ts=1712240781944&live=true

If this isn't the right way to do it let me know. I'd be happy to adjust it according to best practices.

Reply:
I think you might be able to use the namespaced attribute in the dashboard, while keeping the same template variable at the top of the dashboard, like this:

`@argo_rollouts.namespace:$argo_rollouts_namespace.value`

Reply:
Okay! I'll do that then. I edited thanks.